build/test.ps1

```diff
@@ -6,6 +6,7 @@ $all_ok = $True
 
 Write-Host "##[info]Test Native simulator"
 pushd (Join-Path $PSScriptRoot "../src/Simulation/Native/build")
+cmake --build . --config $Env:BUILD_CONFIGURATION
```

Comment:
Out of curiosity, does cmake rebuild in this case if the build has already run?

Reply:
No, cmake checks timestamps and will not build again.